code/api/src/modules/product/query.test.js

```diff
@@ -2,21 +2,109 @@ import request from 'supertest'
 import express from 'express'
 import schema from '../../setup/schema'
 import graphqlHTTP from 'express-graphql'
+import models from "../../setup/models";
+const params = require("../../config/params");
 
 describe('product query', () => {
-  let server;
-
-  beforeAll(() => {
-    server = express();
+  let server = express();
 
+  beforeAll(async () => {
     server.use(
-      "/", 
+      "/",
       graphqlHTTP({
         schema: schema,
-        graphiql: true
+        graphiql: false,
       })
-    )
-  })
+    );
```

Comment:
Love the test setup, Looks like this should solve the issues with tests failing due to existing db seeds.